www/pages/sell/new.tsx

```diff
@@ -122,20 +128,47 @@ function NewSellOrder() {
               />
             </label>
 
-            <label className="flex flex-1 flex-col">
+            <label className="flex-col">
               <div className="font-sans mb-1 text-base">Token</div>
+              <SelectSearch
+                className={(classes: string) => {
+                  return dropDownUI(classes) + ' w-40';
+                }}
```

Comment:
This is fine for the moment, but in the future you should use `classnames`, which is already used in this project. 

```js
import cn from 'classnames';

// ... a little while later

<div
  className={(classes) => cn({ 
     "w-40": true,
     "bg-blue": classes = "input"
  })} 
/>
```